### ISSUE TITLE
Unable to copy/paste/select all in the inspector console

### ISSUE THREAD

**Issue by rmg-x** [LadybirdBrowser/ladybird#2606]
The inspector console textbox doesn't allow you to copy/paste/select all in it.

**Comment by AtkinsSJ** [comment]
Indeed, that's pretty annoying. :sweat_smile: Reproduced with the Qt chrome, I haven't checked AppKit. My guess would be that there's some callback we haven't connected up for the inspector webview.

**Comment by gmta** [comment]
Reopening this to make sure we also deal with AppKit (in)ability to copy/paste/select all :^)

**Comment by rmg-x** [comment]
Closing since the inspector is going to be removed in favor of Firefox dev tools: #3956
